src/fips.c

```diff
@@ -296,7 +296,7 @@ static int FIPSCHECK_verify(const char *path)
 
 	fp = fopen(hmacpath, "r");
 	if (fp == NULL) {
-		rc = 1;
+		rc = 0;
```

Comment:
You could remove this line since `rc` is still at the initial value of 0.

Reply:
Yep, you are right ...

Reply:
done.